### PR TITLE
__apt_mark fixes

### PR DESCRIPTION
**Pull request by 4nd3r**
related #498

### CODE REVIEW

File: cdist/conf/type/__apt_mark/gencode-remote [hunk 1]
```diff
@@ -24,6 +24,20 @@ else
     name="$__object_id"
 fi
 
+apt_version="$(cat "$__object/explorer/apt_version")"
+
+if [ "$apt_version" != '0' ]; then
+    echo 'APT version not supported' >&2
+    exit 0
```
Comment (4nd3r) [inline cdist/conf/type/__apt_mark/gencode-remote]: should we `exit 1` here instead?

File: cdist/conf/type/__apt_mark/gencode-remote [hunk 2]
```diff
@@ -24,6 +24,20 @@ else
...
+fi
+
+package_installed="$(cat "$__object/explorer/package_installed")"
+
+if [ "$package_installed" != '0' ]; then
+    echo "Package $name not installed" >&2
```
Comment (4nd3r) [inline cdist/conf/type/__apt_mark/gencode-remote]: and maybe this message isn't necessary?

File: cdist/conf/type/__apt_mark/gencode-remote (hunk 1 above)
Comment (4nd3r) [inline cdist/conf/type/__apt_mark/gencode-remote]: maybe `exit 1` instead?

File: cdist/conf/type/__apt_mark/gencode-remote (hunk 2 above)
Comment (4nd3r) [inline cdist/conf/type/__apt_mark/gencode-remote]: is this message even necessary?

File: cdist/conf/type/__apt_mark/gencode-remote (hunk 1 above)
Comment (telmich) [inline cdist/conf/type/__apt_mark/gencode-remote]: Ack, exit 1 is important here

**Comment by 4nd3r** [comment]
please merge?

**Comment by darko-poljak** [comment]
@4nd3r I see now that this was PR for beta branch.
This is ok, but I think this should go into master.
Can you create new PR for merging into ungleich/master?
Thanks

**Comment by darko-poljak** [comment]
@4nd3r I merged it from beta into master.
Thanks for your contribution.

**Pull request merged**